src/wallet/wallet.h

```diff
@@ -78,7 +78,8 @@ enum WalletFeature
     FEATURE_WALLETCRYPT = 40000, // wallet encryption
     FEATURE_COMPRPUBKEY = 60000, // compressed public keys
 
-    FEATURE_LATEST = 60000
+    FEATURE_HD = 129900, // Hierarchical key derivation after BIP32 (HD Wallet)
```

Comment:
@MarcoFalke 
If I bump this to 130000 now, HD would no longer work on current master IMO.

For <0.13 incompatibility, we need to stick to the current check based on `CLIENT_VERSION`.
https://github.com/bitcoin/bitcoin/blob/master/src/wallet/walletdb.cpp#L637

Once we change the wallet database format, we could introduce a set of strings about the features the wallet database requires/supports.